src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/Preview.js

```diff
@@ -45,6 +45,22 @@ class Preview extends React.Component<Props> {
     typeDisposer: () => mixed;
     dataDisposer: () => mixed;
 
+    @computed get webspaceKey() {
+        const {
+            router: {
+                attributes: {
+                    webspace,
+                },
+            },
+        } = this.props;
+
+        return webspace;
```

Comment:
this is in case of the SuluArticleBundle undefined and does currently fail.

Beneath (Line 75) we did fallback to the first available webspace

>             this.webspaceKey || this.webspaceOptions[0].value,